### ISSUE TITLE
Post / Story - Excerpt in Document Panel

### ISSUE THREAD

**Issue by jauyong** [GoogleForCreators/web-stories-wp#490]
## Feature Description

As an editor I want to be able to add an excerpt to my Story

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria
1. The excerpt field is in the document panel
2. Editors can add / update the story excerpt

**Comment by jauyong** [comment]
hey @spacedmonkey can you point me to thc rich text formatting that is missing from the Document Panel?
Here is the [figma link](https://www.figma.com/file/Cd9iacVknHK62RO4pRwb68/Stories-Editor?node-id=5837%3A4)

**Comment by swissspidy** [comment]
@jauyong FWIW, in a meeting a while back we decided against using rich text formatting, as that's not something WordPress naturally supports. The excerpt field should be a simple textarea.

For  reference, in Gutenberg it looks like this:

<img width="359" alt="Screenshot 2020-05-04 at 09 38 41" src="https://user-images.githubusercontent.com/841956/80945015-1eb17a00-8deb-11ea-8826-87852bd7ae80.png">

The excerpt is not currently in Figma, but should be trivial to implement a first version even without design.

Needed for things like SEO, so we should add it.

**Comment by jauyong** [comment]
@miina @samitron7 RE: UX Needed, I see we don't have excerpt in figma, but I would imagine it would be the same as the password or permalink field?

![image.png](https://images.zenhubusercontent.com/5e985a7aea3104e101699670/0510fca1-b1bb-4724-9dd7-958611619891)

**Comment by swissspidy** [comment]
It would need to be a multiline text field. Perhaps even with a character count?

**Comment by bmattb** [comment]
@o-fernandez can you confirm if this is needed for Stable.

**Comment by swissspidy** [comment]
My two cents: Given the implications this has for discovery and social sharing, I'd say yes.

**Comment by pbakaus** [comment]
Then we should also change the priority - a P2 in 1.0 stable scope seems weird.

**Comment by o-fernandez** [comment]
@samitron7 we need a text input field in the document settings for the story excerpt. I hope that can be trivial to add - but let me know if you need any more info. 

@swissspidy are there technical requirements for the excerpt? I tried looking in the amp.dev site and couldn't find them. Excerpt length, etc.

**Comment by swissspidy** [comment]
@o-fernandez Nope. The excerpt/description field is not really specific to  AMP and will be mainly used for the [meta description](https://support.google.com/webmasters/answer/35624). There's no limit on how long a meta description can be, but the search result snippets are truncated as needed, typically to fit the device width. Twitter would truncate it after roughly 200 characters it seems.

The stories format or AMP itself doesn't have any requirements for meta descriptions, so you won't find any requirements on amp.dev. And https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/discovery/?format=websites isn't really specific.

Some good documentation about social discovery can also be found at https://developers.google.com/web/fundamentals/discovery/social-discovery

**tl;dr:** We need a simple textarea field to enter the description, like in the screenshots above. Doesn't really need an upper limit I think.

**Comment by samitron7** [comment]
@swissspidy We have design for this [here](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Stable-Release?node-id=1%3A72834). Let me know if you need anything else.


![Screen Shot 2020-08-24 at 3 09 23 PM](https://user-images.githubusercontent.com/10889431/91101400-d0371e00-e61b-11ea-8a37-4eda197d7e9d.png)

**Comment by bmattb** [comment]
Can we move this over to the Sprint Backlog then?

**Comment by o-fernandez** [comment]
Yes, @bmattb let me move it! Added to the sprint (milestone) as well given that's part of the stable release.

**Comment by swissspidy** [comment]
Isn‘t 100 characters too short? As I stated above, Twitter uses 200 for example.

**Comment by csossi** [comment]
Verified in QA (appearing as meta-description as well)
